MyClass.java

```diff
@@ -1,6 +1,16 @@
-public class MyClass {
-    public static void main(String args[]) {
+import java.util.Scanner; //imports scanner class
 
-    }
+public class MyClass {
+	public static void main(String args[]) {
+	   
+	}
+	   public static int pot(){
+	     Scanner potScanner = new Scanner(System.in);
+	     System.out.println("How many people are on the team?");
+	     //"pot" means people on team
+	     int pot = potScanner.nextInt();

```

Comment:
When dealing with user inputs think about what the limits are and if you need to do any error handling.  For example does nextInt() allow negative numbers?